images/pld.yaml

```diff
@@ -0,0 +1,146 @@
+image:
+  distribution: pld
+
+source:
+  downloader: docker-http
+  url: pldlinux/i686:latest
```

Comment:
The `/i686` here would indicate that you're only ever pulling the 32bit image.
If there's a 64bit image, then hopefully just omitting the architecture will do the right thing.

If there's only a 32bit image available, then you shouldn't list amd64 as an architecture in the jenkins job.

Reply:
Yep, this is placeholder actually, as source.url is replaced by -o in jenkins job; as far as I know  distrobuilder is not able to handle template inlining here ({{ architecture }})

Reply:
Ah, right, in this case it's probably best not to set it at all so we don't get confused again when reading it :)

Reply:
Other readers may have no clue what is the image source then, maybe I just comment out this?

Reply:
Yeah, I guess you can comment it out.

For other images, they're almost all lacking the `image.release` property, so folks using the YAMLs without knowing to feed the missing stuff will have issues already :)